CMakeLists.txt

```diff
@@ -38,6 +38,30 @@ set(COMPUTECPP_DEVICE_COMPILER_FLAGS
     "${COMPUTECPP_DEVICE_COMPILER_FLAGS} -no-serial-memop -sycl-compress-name")
 message(STATUS "${COMPUTECPP_DEVICE_COMPILER_FLAGS}")
 
+# Check to see if we've disabled double support in the tests
+option(NO_DOUBLE_SUPPORT "Disable double support when testing." off)
+if(NO_DOUBLE_SUPPORT)
+  # Define NO_DOUBLE_SUPPORT for the host cxx compiler
+  add_definitions(-DNO_DOUBLE_SUPPORT)
+  # Set the computecpp device compiler flags to also define NO_DOUBLE_SUPPORT 
+  list(APPEND COMPUTECPP_DEVICE_COMPILER_FLAGS -DNO_DOUBLE_SUPPORT)
+endif()
+
+# If the user has specified a specific workgroup size for tests, pass that on to the compiler
+if(WGSIZE)
+  add_definitions(-DWGSIZE=${WGSIZE})
+  list(APPEND COMPUTECPP_DEVICE_COMPILER_FLAGS -DWGSIZE=${WGSIZE})
+endif()
+
+# If the user has specified that we should use naive gemm, enable that
+option(NAIVEGEMM "Default to naive GEMM implementations" off)
```

Comment:
seems this desperately needs an _ between naive and gemm?

Reply:
@AdamHarries I fixed code. So you can add _ between the NAIVE and GEMM. Also between the WG and SIZE